deltacat/utils/daft.py

```diff
@@ -0,0 +1,22 @@
+import daft
+from daft.logical.schema import Schema as DaftSchema
+import pyarrow as pa
+
+
+def pyarrow_to_daft_schema(pa_schema: pa.Schema) -> DaftSchema:
+    """Converts a PyArrow schema to Daft schema
+
+    TODO: This should be upstreamed into Daft's Schema class
```

Comment:
Can we address this TODO before merging into DeltaCAT?

Reply:
Addressed!